src/GraphControl/Control/Grapher.cpp

```diff
@@ -699,3 +726,177 @@ namespace GraphControl
         return outputStream;
     }
 }
+
+void Grapher::OnCoreKeyUp(CoreWindow ^ sender, KeyEventArgs ^ e)
+{
+    // We don't want to react to keyboard input unless the graph control has the focus.
+    // NOTE: you can't select the graph control from the mouse for focus but you can tab to it.
+    FrameworkElement ^ whoHasFocus = (FrameworkElement ^) FocusManager::GetFocusedElement();
```

Comment:
Use static_cast here and throughout when casting

Reply:
Changed to us a dynamic_cast like c# "as" functionality.  If we get a nullptr back it isn't the graph control and we skip processing.